src/main/java/org/myrobotlab/service/meta/abstracts/MetaData.java

```diff
@@ -123,23 +123,16 @@
 
   Integer workingLevel = null;
 
-  public MetaData() {
+  public MetaData(String name) {

```

Comment:
so, now metadata is specific to an instance of a service, instead of specific to a service type?

Reply:
Yes - although MavenWrapper and IvyWrapper are consumers of MetaData and do not care what instance they are - as dependency management has no interest.

This revision of Peers and MetaData is vastly smaller and cleaner than the mess I put in initially (back in Manticore).
A very big bonus is the meta data generated with its peer definition is assigned to Service.serviceType (ya the name is vestigial but if I change it, it will break stuff) 
But this information gives the potential for dynamic buttons to be created to manage the lifecycles of the services' peer.
Less code more capability ... good times